docs/setting-up/client/index.md

```diff
@@ -495,3 +496,4 @@ pmm-admin remove <service-type> <service-name>
 [Ubuntu 16.04 (Xenial Xerus)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/xenial/
 [Ubuntu 18.04 (Bionic Beaver)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/bionic/
 [Ubuntu 20.04 (Focal Fossa)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/focal/
+[Ubuntu 22.04 (Jammy Jellyfish)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/jammy/
```

Comment:
We need to remove Xenial (We don't support it anymore) + we need to add Debian 11.

Reply:
ok

Reply:
@nikita-b should we include Ubuntu 11.10 (Oneiric Ocelot)

Reply:
Nope. We support only three Ubuntu:
Bionic, Focal, Jammy